debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -410,6 +410,14 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                                                            .withDependents(DATABASE_WHITELIST_NAME)
                                                            .withDescription("Flag specifying whether built-in tables should be ignored.");
 
+    public static final Field JDBC_DRIVER = Field.create("database.jdbc.driver")
+            .withDisplayName("Jdbc Driver Class Name")
+            .withType(Type.STRING)
```

Comment:
I am running into a test fail with that CLASS and `Validation::isClassName`
{code}
  MySqlConnectorTest.shouldReturnConfigurationDefinition:22->assertConfigDefIsValid:28->lambda$assertConfigDefIsValid$0:40 expected:<['com.mysql.jdbc.Driver']> but was:<[com.mysql.jdbc.Driver]>
{code}
That means I can't supply a default c`om.mysql.jdbc.Driver.class.getName()` any more. If I use the  `com.mysql.jdbc.Driver.class` as default, then `withDefault` method is not designed for class types. Also note, once I do this, then the configuration is going to return a <class> definition not string form of class right? If that is case then this will force me to further changes on look out for class itself not name and classloader to resolve in further downstream code?